src/SelectedMovie/SelectedMovie.js

```diff
@@ -1,16 +1,30 @@
-import movieData from '../movie-data'
 import { IoIosArrowDropleft } from "react-icons/io";
 
-const SelectedMovie = ({ id, goBack }) => {
-    const foundMovie = movieData.movies.find(movie => movie.id === id)
+const SelectedMovie = ({ id, goBack, foundMovie }) => {
     return(
         <article>
             <IoIosArrowDropleft onClick={ () => goBack() } role="button"/>
             <img src={foundMovie.backdrop_path} alt={foundMovie.title + 'image'}/>
             <p>{foundMovie.title}</p>
```

Comment:
Nit-picky, but maybe title is an h1 and the rest are p tags

Reply:
Even nitpickier, there is often only supposed to be a single h1 used in an app, and that's reserved for the app's title. This is a screenreader accessibility convention.